clients/src/main/java/org/apache/kafka/clients/admin/LogDirDescription.java

```diff
@@ -54,11 +64,21 @@ public Map<TopicPartition, ReplicaInfo> replicaInfos() {
         return unmodifiableMap(replicaInfos);
     }
 
+    public OptionalLong totalBytes() {
+        return totalBytes;
+    }
+
+    public OptionalLong usableBytes() {
```

Comment:
Javadoc

Reply:
Also do we want to say something about the contraints on usableBytes and totalBytes (individually and compared with each other).